packages/svelte/src/compiler/phases/3-transform/client/visitors/template.js

```diff
@@ -1314,6 +1314,7 @@ function serialize_event_handler(node, { state, visit }) {
 				binding !== null &&
 				(binding.kind === 'state' ||
 					binding.kind === 'frozen_state' ||
+					binding.declaration_kind === 'import' ||
```

Comment:
Not sure if this has any effect in practise, but I think it would be better to make this `(binding.declaration_kind === 'import' && handler.type === 'MemberExpression')` so that this does not apply the dynamic handler stuff to identifier imports

Reply:
Why not though? Imports can be live bindings, thus dynamic too?

Reply:
That's true, nevermind then